pkg/detectors/lunchmoney/lunchmoney.go

```diff
@@ -0,0 +1,75 @@
+package lunchmoney
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"lunchmoney"}) + `\b([a-zA-Z0-9]{50})\b`)
```

Comment:
suggestion: tighten this regex and use `\b([a-f0-9]{50})\b`

Reply:
I changed the regex and pushed it. Thanks!

Reply:
Some extra context here Mildred, this is a hex-encoded credential which only uses a-f rather than a-z